readme.md

```diff
@@ -8,9 +8,13 @@ Before continuing, install the following tools:
 
 [**Homebrew**](https://brew.sh/): Follow the instructions on this page
 
-[**Python 3.10.0**](https://www.python.org/downloads/macos/): To install python on your local system, you can visit this link and install the relevant version (Python 3.10.0)
+[**Python 3.10.0**](https://www.python.org/downloads/macos/): To install the correct version of python on your local system, check the pipfile and then install the relevant version.
```

Comment:
Can we drop the version number from the link, so we don't need to update the README if the version changes.

Reply:
Oops, I thought I removed it. Will drop that =)